docs/api_specs/02_operator_api_specs.md

```diff
@@ -0,0 +1,86 @@
+# Child chain
```

Comment:
we might want to add `settle` which will take a transaction and RC-exchange settlement proof, to start designing around that.

Then we'll probably mirror that in `Watcher`, just like we're doing with `submit`